go/worker/common/p2p/peermgmt.go

```diff
@@ -92,7 +92,11 @@ func (mgr *PeerManager) isPeerAuthorized(id core.PeerID) bool {
 	mgr.RLock()
 	defer mgr.RUnlock()
 
-	return mgr.peers[id] != nil
+	// TODO: if we're txnscheduler anyone can publish, unless we would use a
```

Comment:
Hm, right, this makes it more likely that random people can mess with libp2p. But I guess that's what the (not yet implemented) P2P proxy/sentry is for. @Yawning any thoughts?

Reply:
Oh gross.  What's "all nodes" in this context, the entire internet?

Reply:
Yeah, so previously it was "the entire internet can connect to txnscheduler gRPC" now it is "the entire internet can connect to libp2p". It is more scary I guess, but not sure how it could be avoided as there needs to be a way for anyone to submit transactions?

Reply:
Yeah, well technically the nodes could still limit access on a different layer to trusted "client nodes" (e.g. firewalling ... would in any case make sense at least in the case of permissioned runtimes - but could do the same for others as well) - i guess the only way to avoid that would be having clients also needing to register.

Reply:
And everything available publicly should run behind proxies (we don't have libp2p sentries yet though).